tee-worker/enclave-runtime/src/utils.rs

```diff
@@ -94,50 +98,76 @@ pub(crate) fn get_triggered_dispatcher(
 pub(crate) fn get_validator_accessor_from_solo_or_parachain(
 ) -> Result<Arc<EnclaveValidatorAccessor>> {
 	let validator_accessor =
-		if let Ok(solochain_handler) = GLOBAL_FULL_SOLOCHAIN_HANDLER_COMPONENT.get() {
+		if let Ok(solochain_handler) = GLOBAL_INTEGRITEE_SOLOCHAIN_HANDLER_COMPONENT.get() {
```

Comment:
Maybe we need to change them to Litentry version => multiple places

Reply:
Yes. Already done.